@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -254,15 +240,14 @@
   position: absolute;
   left: 0;
   right: 0;
-  z-index: var(--a-z-index-popover);
-  top: calc(100% + var(--a-spacing-2));
-  border: 1px solid var(--ac-combobox-list-border-color, var(--a-border-divider));
+  z-index: var(--ax-z-index-popover);
```

Comment:
We will remove z-index tokens i new setup, so can hardcode this to a large value